frontend/src/pages/Graph/GraphPage.tsx

```diff
@@ -632,7 +632,7 @@ class GraphPageComponent extends React.Component<GraphPageProps, GraphPageState>
       return;
     }
 
-    if (event.hasMissingSC) {
+    if (event.isOutOfMesh && !event.isAmbient) {
       AlertUtils.add(
         `A node with a missing sidecar provides no node-specific telemetry and can not provide a node detail graph.`,
         undefined,
```

Comment:
Do we need the `isAmbient` check here?  If the node is outOfMesh then wouldn't this apply to ambient or non-ambient?

Reply:
Yes, that is right. Removed isAmbient check